src/scenic/syntax/ast.py

```diff
@@ -30,7 +55,7 @@ def __init__(
     ) -> None:
         super().__init__(*args, **kwargs)
         self.className = className
-        self.specifiers = specifiers
+        self.specifiers = specifiers if specifiers is not None else []
```

Comment:
Because specifiers can be empty, and the grammar rule generates `None` in such cases, we need to manually set it to the empty list if `specifiers` is None